### DISCUSSION TITLE
New Destination: Local Parquet

### DISCUSSION THREAD

**Discussion by ucatbas** [airbytehq/airbyte#36082] [New Connector Request]
## New Connector: Local Parquet Destination
* Destination - Local [Parquet](https://github.com/apache/parquet-mr)

## Describe the context around this new connector
* Parquet is a column-oriented data file format designed for efficient data storage and retrieval.
* It will be useful for on-premise users.

## Describe the alternative you are considering or using
* Probably [Apache Arrow](https://arrow.apache.org/docs/python/parquet.html)

**Comment by gcleaves** [comment]
Would love to see this
